SampleApp/SearchPinsViewController.swift

```diff
@@ -50,7 +57,15 @@ class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate {
     present(alert, animated: true, completion: nil)
   }
 
-  // MARK:- private
-
+  func selected(_ location: PeliasMapkitAnnotation) {
+    print("Selected \(String(describing: location.title))")
+    searchField.text = location.title
+    location.setTarget(target: self, action: #selector(self.annotationClicked(annotation:)))
+    try? self.removeAnnotations()
+    try? self.add([location])
 
+    animate(toZoomLevel: 10, withDuration: 1.0)
```

Comment:
Easy enough change to make, although to be honest I thought through some of that (being zoomed in further specifically) and honestly it didn't seem that jarring to reset the view. That said, again it's a demo app so I only put minimal thought into the UX haha :D